src/database/models/vacationRequest.ts

```diff
@@ -19,12 +20,13 @@ interface VacationRequestModel {
 }
 
 /**
- * Enumerator for the vacation request statuses
+ * DynamoDB model for vacation request status
  */
-enum VacationRequestStatuses {
-  PENDING = "PENDING",
-  APPROVED = "APPROVED",
-  REJECTED = "DECLINED"
+interface VacationRequestStatusesModel {
```

Comment:
Do we need this model though, shouldn't this be described in the spec and then we can just reference the spec. Sometimes the spec and a DB model differ but in this case I don't think there is any reason for them so we can just have a single source defining the status type.